### DISCUSSION TITLE
[Chapter 11] NTPD 란?

### DISCUSSION THREAD

**Discussion by JoisFe** [cowaine/spring-boot-for-msa#180] [Chapter 11. Spring Scheduling Tasks] [answered]
11.3 에서 두 대 이상의 배치 서버를 동시에 실행한 배치 프로세스 경우 공유 자원에 데드 락을 발생할 수 있어 데드락 회피 방법으로
`모든 서버군의 서버는 NTPD를 사용하여 서버 시간을 동기화한다` 라고 되어있습니다.

여기서 **NTPD**가 무엇인지 아시는 분 있나요?

**Comment by Irisation23** [comment]
`NYPD`가 뉴욕 폴리스인건 알고있는데...

**Comment by corock** [comment, marked as answer]
저도 처음 본 건데 찾아보니 NTPd(_Network Time Protocol daemon_) 의미가 있네요.

> NTPd 은 리눅스 운영체제의 데몬 프로세스 중 하나로, NTP 프로토콜을 구현하여 로컬 시스템과 시계 원본 서버 간의 시간을 교정하는 데 사용된다.

사실 NTP 프토토콜도 처음 보는데, NTP(_Network Time Protocol_) 도 찾아봤습니다.

> NTP 는 지연이 있을 수 있는 네트워크로 구성된 환경에서 시스템 간의 시간을 동기화하기 위한 규약을 말한다.
